imports/plugins/included/social/client/components/facebook.js

```diff
@@ -88,7 +88,9 @@ class FacebookSocialButton extends Component {
     });
 
     return (
-      <a className="btn btn-flat facebook-share" href="#" onClick={this.handleClick} target="_blank">
+      <a className="btn btn-flat facebook-share" aria-label="Share to Facebook" href="#" onClick={this.handleClick}
```

Comment:
Generally if you are using a href="#" the a tag should probably be a button tag as you aren't 'linking' or 'anchoring' to another part of the page!